themes/classic/modules/ps_customeraccountlinks/ps_customeraccountlinks.tpl

```diff
@@ -39,13 +39,31 @@
     </span>
   </div>
   <ul class="account-list collapse" id="footer_account_list">
-    {foreach from=$my_account_urls item=my_account_url}
-        <li>
-          <a href="{$my_account_url.url}" title="{$my_account_url.title}" rel="nofollow">
-            {$my_account_url.title}
-          </a>
-        </li>
-    {/foreach}
-    {hook h='displayMyAccountBlock'}
+    {if $customer.is_logged}
+        <li><a href="{$urls.pages.identity}" title="{l s='Information' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Information' d='Shop.Theme.Customeraccount'}</a></li>
+        {if $customer.addresses|count}
+          <li><a href="{$urls.pages.addresses}" title="{l s='Addresses' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Addresses' d='Shop.Theme.Customeraccount'}</a></li>
+        {else}
+          <li><a href="{$urls.pages.address}" title="{l s='Add first address' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Add first address' d='Shop.Theme.Customeraccount'}</a></li>
+        {/if}
+        {if !$configuration.is_catalog}
+          <li><a href="{$urls.pages.history}" title="{l s='Orders' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Orders' d='Shop.Theme.Customeraccount'}</a></li>
+        {/if}
+        {if !$configuration.is_catalog}
+          <li><a href="{$urls.pages.order_slip}" title="{l s='Credit slips' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Credit slips' d='Shop.Theme.Customeraccount'}</a></li>
+        {/if}
+        {if $configuration.voucher_enabled && !$configuration.is_catalog}
+          <li><a href="{$urls.pages.discount}" title="{l s='Vouchers' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Vouchers' d='Shop.Theme.Customeraccount'}</a></li>
+        {/if}
+        {if $configuration.return_enabled && !$configuration.is_catalog}
+          <li><a href="{$urls.pages.order_follow}" title="{l s='Merchandise returns' d='Shop.Theme.Customeraccount'}" rel="nofollow">{l s='Merchandise returns' d='Shop.Theme.Customeraccount'}</a></li>
+        {/if}
+        {hook h='displayMyAccountBlock'}
```

Comment:
Have to be outside the is_logged condition, wdyt?

Reply:
@PierreRambaud Depends, but I was thinking that all the hooked modules are related to a logged in customer. 🤔 We can move it out to preserver original behavior, but modules should adapt. Or we just keep it there.

Reply:
We have to keep the original behavior, modules have to adapt if they need something specific :)

Reply:
@PierreRambaud Agree :-)

I kept the hook twice, so we have it above "Sign out" when logged in. They can never run twice. OK?